docs/ppl-zkp-design.md

```diff
@@ -0,0 +1,125 @@
+# ZKP Designs for Transactions
+
+-- Navin Kabra
+
+_Status: This is an early, unreviewed draft. It is likely to have errors._
+
+## UTXO based design
+
+We assume that a wallet consists of multiple sleeves. Each sleeve holds value stores of only one type of IOUType and UOM.
+
+
+
+### Sleeve
+
+A sleeve consists of:
+
+- WalletID
+- SleeveID
+- IOUType
+- UOM (Unit of Measurement)
+- PublicKey
+- NotaryID (the primary notary for this wallet)
```

Comment:
Should we keep the primary notary to be decided at run time by the notary pool?

Reply:
Actually, based on our discussion regarding how Notaries are assigned and possibly changed, I'll need to make bunch of small changes throughout the document. Will do that next week.